### DISCUSSION TITLE
"field archive not found in type config.Project"

### DISCUSSION THREAD

**Discussion by natefinch** [goreleaser/goreleaser#2760] [Q&A] [answered]
I went to release Mage and goreleaser is giving me an error about archive not being valid.

```
   • releasing...     
   • loading config file       file=.goreleaser.yml
   ⨯ release failed after 0.00s error=yaml: unmarshal errors:
  line 29: field archive not found in type config.Project
```

Did this change at some point? I installed latest of goreleaser:

```
% goreleaser --version
goreleaser version dev
module version: v1.1.0, checksum: h1:YySqqYTX9kxRU0e/fGQrhivXk8/zD4iUOlL7h0I2gBk=
```

Here's my yaml: 
https://github.com/magefile/mage/blob/07afc7d24f4d6d6442305d49552f04fbda5ccb3e/.goreleaser.yml#L29

Thanks in advance!

**Comment by caarlos0** [comment, marked as answer]
yes, long ago, it's pluralized now

- https://goreleaser.com/customization/archive/
- https://goreleaser.com/deprecations/#archive
